app/helpers/active_admin_helpers.rb

```diff
@@ -0,0 +1,90 @@
+module ActiveAdminHelpers
+  def self.all_time_periods
+    TimePeriod.all
+  end
+
+  def self.time_period_vars(team: nil, time_period: nil, previous_time_period: nil, current_period: nil)
+    all_time_periods = ActiveAdminHelpers.all_time_periods
```

Comment:
why we need separate method? you use `all_time_periods` method just in `time_period_vars` so it seems like it should be local variable
`all_time_periods = TimePeriod.all`

Reply:
I can't see changes here